src/components/Shortcuts/ClipboardShortcuts/ClipboardShortcuts.tsx

```diff
@@ -61,19 +62,23 @@ export function ClipboardShortcuts() {
         return;
       }
 
-      const activeElementId = slideInstance.getActiveElementId();
+      const activeElementIds = slideInstance.getActiveElementIds();
 
-      if (!activeElementId) {
+      if (!activeElementIds.length) {
```

Comment:
it is much common across code to use `length`:
```suggestion
      if (activeElementIds.length === 0) {
```